x-pack/test/security_solution_api_integration/test_suites/lists_and_exception_lists/lists_items/trial_license_complete_tier/lists/delete_lists.ts

```diff
@@ -35,14 +35,15 @@ export default ({ getService }: FtrProviderContext) => {
   const log = getService('log');
   const utils = getService('securitySolutionUtils');
 
-  describe('@ess @serverless @serverlessQA delete_lists', () => {
+  // After adding missing `await` delete list request fails with response code 200, not 409
+  describe.skip('@ess @serverless @serverlessQA delete_lists', () => {
```

Comment:
without `await` test was finishing before API call is completed, false positive?

Reply:
Does the entire suite need to be skipped, or just the individual test? Currently, they're both skipped, which is confusing.